pkg/apis/ceph.rook.io/v1/types.go

```diff
@@ -545,9 +545,11 @@ type ClientSpec struct {
 }
 
 type CleanupPolicySpec struct {
-	DeleteDataDirOnHosts string `json:"deleteDataDirOnHosts"`
+	CleanupConfirmation CleanupConfirmationProperty `json:"confirmation,omitempty"`
```

Comment:
Why name the property different from the yaml name?
```suggestion
	Confirmation CleanupConfirmationProperty `json:"confirmation,omitempty"`
```